packages/@n8n/benchmark/scripts/runForN8nSetup.mjs

```diff
@@ -36,6 +36,14 @@ async function main() {
 	const vus = argv.vus;
 	const duration = argv.duration;
 
+	const hasN8nLicense = !!n8nLicenseCert || !!n8nLicenseActivationKey;
+	if (n8nSetupToUse === 'scaling' && !hasN8nLicense) {
+		console.error(
+			'N8n license is required to run the scaling setup. Please provide N8N_LICENSE_CERT or N8N_LICENSE_ACTIVATION_KEY',
```

Comment:
```suggestion
			'n8n license is required to run the scaling setup. Please provide N8N_LICENSE_CERT or N8N_LICENSE_ACTIVATION_KEY',
```

Reply:
we always refer to our company as "n8n" (all lower-case)

Reply:
Tru dat, copilot should learn that! 😄